src/backend/base/langflow/services/settings/base.py

```diff
@@ -181,6 +181,8 @@ class Settings(BaseSettings):
     """The maximum number of transactions to keep in the database."""
     max_vertex_builds_to_keep: int = 3000
```

Comment:
Does a `WHERE ... IN (...)` works with 3000 entries in SQlite and Postgres ?

Reply:
That's a good point. It should but could be inefficient. Any suggestion?

Reply:
Actually it's not the number of entries we'll get in the `where`. So my comment is irrelevant.